lua-mosquitto.c

```diff
@@ -1273,37 +1863,43 @@ static const struct luaL_Reg R[] = {
 };
 
 static const struct luaL_Reg ctx_M[] = {
-	{"destroy",			ctx_destroy},
-	{"__gc",			ctx_destroy},
-	{"reinitialise",	ctx_reinitialise},
-	{"will_set",		ctx_will_set},
-	{"will_clear",		ctx_will_clear},
-	{"login_set",		ctx_login_set},
+	{"destroy",				ctx_destroy},
+	{"__gc",				ctx_destroy},
+	{"reinitialise",		ctx_reinitialise},
+	{"will_set",			ctx_will_set},
+	{"will_set_v5",			ctx_will_set_v5},
+	{"will_clear",			ctx_will_clear},
+	{"login_set",			ctx_login_set},
 	{"tls_insecure_set",	ctx_tls_insecure_set},
-	{"tls_set",		ctx_tls_set},
-	{"tls_psk_set",		ctx_tls_psk_set},
-	{"tls_opts_set",	ctx_tls_opts_set},
-	{"threaded_set",	ctx_threaded_set},
-	{"option",		ctx_option},
-	{"connect",			ctx_connect},
-	{"connect_async",	ctx_connect_async},
-	{"reconnect",		ctx_reconnect},
-	{"reconnect_async",	ctx_reconnect_async},
-	{"disconnect",		ctx_disconnect},
-	{"publish",			ctx_publish},
-	{"subscribe",		ctx_subscribe},
-	{"unsubscribe",		ctx_unsubscribe},
-	{"loop",			ctx_loop},
-	{"loop_forever",	ctx_loop_forever},
-	{"loop_start",		ctx_loop_start},
-	{"loop_stop",		ctx_loop_stop},
-	{"socket",			ctx_socket},
+	{"tls_set",				ctx_tls_set},
+	{"tls_psk_set",			ctx_tls_psk_set},
+	{"tls_opts_set",		ctx_tls_opts_set},
+	{"threaded_set",		ctx_threaded_set},
+	{"option",				ctx_option},
+	{"connect",				ctx_connect},
+	{"connect_bind_v5",		ctx_connect_bind_v5},
+	{"connect_async",		ctx_connect_async},
+	{"reconnect",			ctx_reconnect},
+	{"reconnect_async",		ctx_reconnect_async},
+	{"disconnect",			ctx_disconnect},
+	{"disconnect_v5",		ctx_disconnect_v5},
+	{"publish",				ctx_publish},
+	{"publish_v5",			ctx_publish_v5},
+	{"subscribe",			ctx_subscribe},
+	{"subscribe_v5",		ctx_subscribe_v5},
+	{"unsubscribe",			ctx_unsubscribe},
+	{"unsubscribe_v5",		ctx_unsubscribe_v5},
+	{"loop",				ctx_loop},
+	{"loop_forever",		ctx_loop_forever},
+	{"loop_start",			ctx_loop_start},
+	{"loop_stop",			ctx_loop_stop},
+	{"socket",				ctx_socket},
```

Comment:
all the whitespace changes here make it harder to follow :|

Reply:
I tried to revert the whitespace.